### DISCUSSION TITLE
GitHub Pages

### DISCUSSION THREAD

**Discussion by Coolkidllama1** [community/community#6911] [Other Feature Feedback, Questions, & Ideas]
Can y'all add a feature for GitHub desktop for when I push origin for after its done it will bring up a loading bar on the progress of the changes actually applying online like if I made another website inside my repository or something. After that feature is released then try to add time predictions but I just want the loading bar first. Please I need this because I thought something was wrong with GitHub when I copied files into a new folder and made a whole new thing and GitHub pages wasn't detecting it. Also there should be a GitHub pages section in feedback lol.

**Comment by jfullstackdev** [comment]
if you will be committing that directly on the remote branch, then you might not be able to notice that,

the correct process is always to create a branch and push the changes there so you can create a Pull Request, that way, unintentional changes will not happen
